src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -672,343 +464,132 @@ subroutine Run ( GC, IMPORT, EXPORT, CLOCK, RC )
          _VERIFY(STATUS)
          ALLOCATE(VCr8(is:ie,js:je,lm), STAT=STATUS); 
          _VERIFY(STATUS)
-         ALLOCATE(PLEr8(is:ie,js:je,lm+1), STAT=STATUS); 
-         _VERIFY(STATUS)
-
+         
          ! Prepare inputs to fv_computeMassFluxes
-
-         ! Copy UA,VA to UC,VC while handling top-down vs. bottom-up meteorological data
```

Comment:
The explanation for flipping is removed here. Keeping it with modification would help make the code more understanding. This comment extends to all parts of the file that flip vertically.